### PR TITLE
Add new syntax 'ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n'

### PR DESCRIPTION
**Pull request by lyzx2001**
close https://github.com/pingcap/docs/issues/9698
Add syntax 'ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n', which can retrieve DDL commands' content within a certain range (n+1, n+m).

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
A new syntax is added.
Add description for syntax 'ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n', which can retrieve DDL commands' content within a certain range (n+1, n+m).

I added a new syntax 'ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n', and the details can refer to:
https://github.com/pingcap/tidb/pull/36285
https://github.com/pingcap/tidb/pull/36480

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
/
- Other reference link(s):
https://github.com/pingcap/tidb/pull/36285
https://github.com/pingcap/tidb/pull/36480

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 1]
```diff
@@ -113,6 +113,56 @@ ADMIN SHOW DDL JOB QUERIES 51;
 
 只能在 DDL 历史作业队列中最近十条结果中搜索与 `job_id` 对应的正在运行中的 DDL 作业。
 
+### `ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n`
+
+`ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n` 语句用于查看指定范围 `(n+1, n+m)` 的 `job_id` 对应的 DDL 任务的原始 SQL 语句：
```
Comment (xhebox) [inline sql-statements/sql-statement-admin-show-ddl.md:118]: 不是`[n,n+m]`吗？

**Review by xhebox — COMMENTED** [review]
上面语法图没改

File: sql-statements/sql-statement-admin-show-ddl.md (hunk 1 above)
Comment (lyzx2001) [reply sql-statements/sql-statement-admin-show-ddl.md:118]: 我之前举的例子有点容易搞混，改了一下，范围的话我指的是从n+1到n+m，两头都是包含，我写的(n+1, n+m)是想表达这个意思，现在改成[n+1, n+m]，不知道是不是你这边说的意思

Comment (lyzx2001) [reply sql-statements/sql-statement-admin-show-ddl.md]: 这三句带limit的语法与MySQL带limit的语法一致
https://dev.mysql.com/doc/refman/8.0/en/select.html

**Review by xhebox — APPROVED** [review]

File: sql-statements/sql-statement-admin-show-ddl.md (hunk 1 above)
Comment (xhebox) [reply sql-statements/sql-statement-admin-show-ddl.md]: OK的

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 2]
```diff
@@ -113,6 +113,57 @@ ADMIN SHOW DDL JOB QUERIES 51;
...
+### `ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n`
+
+`ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n` 语句用于查看指定范围 `[n+1, n+m]` 的 `job_id` 对应的 DDL 任务的原始 SQL 语句：
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL JOB QUERIES LIMIT m;  # Retrieve first m rows
+ADMIN SHOW DDL JOB QUERIES LIMIT n, m;  # Retrieve rows [n+1, n+m]
+ADMIN SHOW DDL JOB QUERIES LIMIT m OFFSET n;  # Retrieve rows [n+1, n+m]
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-show-ddl.md:125]: 注释建议用中文，并以 -- 开头

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 3]
```diff
@@ -113,6 +113,57 @@ ADMIN SHOW DDL JOB QUERIES 51;
...
+```
+
+这里 n 和 m 都是非负整数。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-show-ddl.md:128]:
```suggestion
以上语法中 `n` 和 `m` 都是非负整数。语法的具体示例如下：
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 4]
```diff
@@ -113,6 +113,57 @@ ADMIN SHOW DDL JOB QUERIES 51;
...
+| JOB_ID | QUERY                                  | 
++--------+----------------------------------------+
+|     54 | DROP TABLE IF EXISTS t3                |
+|     53 | ALTER TABLE t1 DROP INDEX index1       |
+|     52 | ALTER TABLE t1 ADD INDEX index1 (col1) | 
++--------+----------------------------------------+
+3 rows in set (0.00 sec)
+```
+
+可以在 DDL 历史作业队列任意指定范围中搜索与 `job_id` 对应的正在运行中的 DDL 作业，没有 `ADMIN SHOW DDL JOB QUERIES` 语句的最近十条结果限制。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-show-ddl.md:165]:
```suggestion
该语句可以在 DDL 历史作业队列任意指定范围中搜索与 `job_id` 对应的正在运行中的 DDL 作业，没有 `ADMIN SHOW DDL JOB QUERIES` 语句的最近 10 条结果的限制。
```

File: sql-statements/sql-statement-admin-show-ddl.md (hunk 2 above)
Comment (lyzx2001) [reply sql-statements/sql-statement-admin-show-ddl.md]: 已修改

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**